test/src/Makefile

```diff
@@ -18,6 +18,7 @@ test: init
 	go mod download
 	go test -v -timeout 5m -run TestExamplesComplete
 	go test -v -timeout 5m -run TestExamplesStacks
+	go test -v -timeout 5m -run TestExamplesSpacelift
```

Comment:
Consider switching to `go test -v -timeout 15m ./...` and adding `t.Parallel()` to the tests. That way we won't miss any (there are already more than these 3 that aren't running because we've singled them out) and they can run in parallel.

Reply:
can do in the next PR, thanks.
Anyway, those test don't provision anything, so they are fast enough

Reply:
5m should be ok to process YAML files :)